<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>system manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_ID">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_mode">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>관리자 메뉴</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tab_manage">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>사용자 관리 (&amp;1)</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lb_user_id">
               <property name="text">
                <string>사용자 ID   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_user_id">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="lb_grant">
                 <property name="text">
                  <string>사용자 권한</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="com_grant"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lbl_pass">
               <property name="text">
                <string>password  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_pass"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>사용자 이름</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_name"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>e-mail       </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_email"/>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="btn_insert_user">
               <property name="text">
                <string>유저 추가(&amp;i)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_edit_user">
               <property name="text">
                <string>유저 편집 (&amp;e)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_del_user">
               <property name="text">
                <string>유저 삭제 (&amp;d)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QTableWidget" name="table_user">
             <property name="columnCount">
              <number>4</number>
             </property>
             <column/>
             <column/>
             <column/>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>시스템 관리(&amp;2)</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_init">
               <property name="text">
                <string>초기화(&amp;i)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_backup">
               <property name="text">
                <string>백업(&amp;b)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_restore">
               <property name="text">
                <string>복구(&amp;r)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_exit">
         <property name="text">
          <string>종료 (&amp;x)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
